pkg/config/config.go

```diff
@@ -264,6 +264,10 @@ type RuntimeHandler struct {
 	// Output of the "features" subcommand.
 	// This is populated dynamically and not read from config.
 	features runtimeHandlerFeatures
+
+	// Inheritance request
+	// Fill in the Runtime information (paths and type) from the default runtime
+	UseDefaultRuntime bool `toml:"use_default_runtime,omitempty"`
```

Comment:
I'm just a bit concerned about the naming, how about `inherit_default_options` or similar? Maybe other @cri-o/cri-o-maintainers have other thoughts on that.

Reply:
Is inherit_default_runtime OK? Because it only replaces the runtime_* variables.